server/proto/types.go

```diff
@@ -1,3 +1,12 @@
 package ChannerProto
 
-type UUID uint64
+import (
+	"github.com/umegaya/yue"
+)
+
+type UUID yue.UUID
```

Comment:
__[gometalinter]__
* [gotype] undeclared name: yue
* [golint] exported type UUID should have comment or be unexported